mx.py

```diff
@@ -911,10 +911,10 @@ def siblings_dir(suite_dir):
             _, primary_vc_root = VC.get_vc_root(suite_dir, abortOnError=False)
             if not primary_vc_root:
                 suite_parent = dirname(suite_dir)
-                # Use the heuristic of a 'ci.hocon' file being
+                # Use the heuristic of a '.mx_vcs_root' file being
                 # at the root of a repo that contains multiple suites.
-                hocon = join(suite_parent, 'ci.hocon')
-                if exists(hocon):
+                mx_vcs_root = join(suite_parent, '.mx_vcs_root')
```

Comment:
during internal review, it came up that we should rather check for both files (`ci.hocon` or `.mx_vcs_root`) to avoid any breaking change to thing that might have relied on the existing `ci.hocon` heuristic.

Reply:
Thanks @gilles-duboscq !
This should now be resolved.